app/api/v1/defaults.rb

```diff
@@ -36,7 +36,7 @@ module Defaults
 
         # Global handler for any unexpected exception
         rescue_from :all do |e|
-          Airbrake.notify(e)
+          env['rack.exception'] = e # Store it in Rack env for Airbrake middleware
```

Comment:
Question: the `Airbrake.notify` method is used in other places, too. In one case it sends a payload alongside the exception, in another it sends an error message without the actual exception. How are those two cases supposed to work now? Or is it OK to continue using `Airbrake.notify` still?

Reply:
`Airbrake.notify` can still be called in any scenario, no problem or conflict at all.

If you want to include request metadata in the notification you can use Rails' `notify_airbrake` helper as reference: https://github.com/airbrake/airbrake/blob/68bec4972aa3bbc75014f8f67ea7ea5213b0a02c/lib/airbrake/rails/controller_methods.rb#L31